packages/core/src/components/people/repository-postgres.ts

```diff
@@ -30,9 +36,49 @@ export class PeopleRepositoryPostgres implements PeopleRepository {
     };
   }
 
-  async findAll(): Promise<Person[]> {
-    const dbPeople = await this.db.select().from(people);
-    return dbPeople.map(this.mapToDomainPerson);
+  async findAll(
+    config: PeopleFindAllConfig = { pagination: { page: 1, pageSize: 10 } },
+  ): Promise<PaginatedResponse<Person>> {
+    const {
+      pagination: { page, pageSize, filter },
+    } = config;
+
+    const offset = (page - 1) * pageSize;
```

Comment:
this seems to be why the tests fail

Reply:
Yep, on it. I need to setup separate branches for testing 🙏

Reply:
Ah, no. It's a type issue.